TradeItIosTicketSDK2/TradeItPortfolioPositionsTableViewManager.swift

```diff
@@ -91,22 +91,13 @@ class TradeItPortfolioPositionsTableViewManager: NSObject, UITableViewDelegate,
     func tableView(tableView: UITableView, editActionsForRowAtIndexPath indexPath: NSIndexPath) -> [UITableViewRowAction]? {
         let buyAction = UITableViewRowAction(style: .Normal, title: "BUY") { action, index in
             let position = self.positions[index.row]
-            let order = TradeItOrder()
-            order.action = TradeItOrderAction.Buy
-            order.linkedBrokerAccount = position.linkedBrokerAccount
-            order.symbol = position.position?.symbol
-
-            self.delegate?.buyButtonWasTappedWith(order: order)
+            self.delegate?.tradeButtonWasTapped(forPortFolioPosition: position, orderAction: TradeItOrderAction.Buy)
         }
         buyAction.backgroundColor = UIColor.greenColor()
         
         let sellAction = UITableViewRowAction(style: .Normal, title: "SELL") { action, index in
             let position = self.positions[index.row]
-            let order = TradeItOrder()
-            order.action = TradeItOrderAction.Sell
-            order.linkedBrokerAccount = position.linkedBrokerAccount
-            order.symbol = position.position?.symbol
-            self.delegate?.sellButtonWasTappedWith(order: order)
+            self.delegate?.tradeButtonWasTapped(forPortFolioPosition: position, orderAction: TradeItOrderAction.Sell)
```

Comment:
You don't need the whole `TradeItOrderAction.Sell` bit. You can just use `.Sell` or `.Buy` etc.